vms/job/services.py

```diff
@@ -81,6 +80,31 @@ def get_jobs_by_event_id(e_id):
     return job_list
 
 
+def get_jobs_by_date(start_date, end_date):
+    """
+    filters jobs on the basis of start date and end date
+    :param start_date: starting date of job
+    :param end_date: ending date of job
+    :return: list of filtered jobs
+
+    """
+    is_valid = True
+    result = None
+    kwargs = {}
+    if start_date:
+        kwargs['start_date__gte'] = start_date
+    if end_date:
+        kwargs['start_date__lte'] = end_date
```

Comment:
`kwargs['start_date__lte'] = end_date`. Here, you are filtering jobs with `start_date` less than the `end_date provided in the form`. Was this intentional?

Reply:
I think in this case, if I provide an end date I expect the results to contain all jobs with end date less than that and after today.